### PR TITLE
LINK-1754 | Modal before reservation time is expiring

### PR DESCRIPTION
**Pull request by jorilindell**
## Description
- Display accessibility notification after loading reservation timer. Text tells to user how much time there is to register.
- Display modal if registration time is expiring soon

## Closes
[LINK-1754](https://helsinkisolutionoffice.atlassian.net/browse/LINK-1754)

## Screenshot
Screenshot 2024-04-11 at 13 20 53

[LINK-1754]: https://helsinkisolutionoffice.atlassian.net/browse/LINK-1754?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: public/locales/fi/reservation.json [hunk 1]
```diff
@@ -1,7 +1,20 @@
...
     "title": "Varausaika on täynnä."
-  }
+  },
+  "reservationTimeExpiringModal": {
+    "buttonClose": "Sulje",
+    "text": "Varauksen aika on päättymässä {{count}} sekunnin päästä",
+    "text_other": "Varauksen aika on päättymässä {{count}} sekunnin päästä",
+    "title": "Varauksen aika on päättymässä"
+  },
+  "second": "{{count}} sekuntti",
```
Comment (harriris-vincit) [inline public/locales/fi/reservation.json:17]:
```suggestion
  "second": "{{count}} sekunti",
```

File: public/locales/fi/reservation.json [hunk 2]
```diff
@@ -1,7 +1,20 @@
...
+  "second_other": "{{count}} sekunttia",
```
Comment (harriris-vincit) [inline public/locales/fi/reservation.json:18]:
```suggestion
  "second_other": "{{count}} sekuntia",
```

File: public/locales/en/reservation.json [hunk 3]
```diff
@@ -1,7 +1,20 @@
 {
+  "hour": "{{count}} hour",
+  "hour_other": "{{count}} hours",
+  "minute": "{{count}} minutes",
```
Comment (harriris-vincit) [inline public/locales/en/reservation.json:4]:
```suggestion
  "minute": "{{count}} minute",
```
This should probably also use the singular form?

File: public/locales/fi/reservation.json (hunk 1 above)
Comment (jorilindell) [reply public/locales/fi/reservation.json]: Fixed

File: public/locales/fi/reservation.json (hunk 2 above)
Comment (jorilindell) [reply public/locales/fi/reservation.json]: Fixed

File: public/locales/en/reservation.json (hunk 3 above)
Comment (jorilindell) [reply public/locales/en/reservation.json]: Fixed

**Review by harriris-vincit — APPROVED** [review]
Looks good, and worked as expected when testing :rocket:

**Pull request merged**